src/nfs/nfs_client_impl.cpp

```diff
@@ -572,7 +572,7 @@ void nfs_client_impl::register_cli_commands()
     static std::once_flag flag;
     std::call_once(flag, [&]() {
         _nfs_max_copy_rate_megabytes_cmd = dsn::command_manager::instance().register_command(
-            {"nfs.max_copy_rate_megabytes"},
+            {"nfs.max_copy_rate_megabytes_per_disk"},
```

Comment:
I recommend adding this update in next pull request, because they don't have relationship with tracer, but relate to disk throttle.

Reply:
done